composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "df2bfe94e9160251d0a853ee3531942a",
+    "content-hash": "e10a62ff80f933f1f186f448e92a0229",
```

Comment:
I think we should revert this change

Reply:
This is the result of running `composer update` with the new content in `composer.json`. It's correct if we're still modifying `composer.json` in this pull request.